charts/fluent-bit-out-hsdp/values.yaml

```diff
@@ -1,424 +1,26 @@
-        Wildcard            pod_name
-        Operation lift
-        Nested_under kubernetes
-        Add_prefix   kubernetes_
-
-    [FILTER]
-        Name modify
-        Match kube.*
-        Hard_copy log logdata_message
-
-    [FILTER]
-        Name modify
-        Match kube.*
-        Hard_copy kubernetes_host server_name
-
-    [FILTER]
-        Name modify
-        Match kube.*
-        Hard_copy kubernetes_pod_name app_instance
-
-    [FILTER]
-        Name modify
-        Match kube.*
-        Hard_copy kubernetes_container_name app_name
-
-    [FILTER]
-        Name modify
-        Match kube.*
-        Hard_copy kubernetes_namespace_name component
-
-    [FILTER]
-        Name modify
-        Match kube.*
-        Hard_copy kubernetes_container_image app_version
-
-
-  ## https://docs.fluentbit.io/manual/pipeline/outputs
-  outputs: |
-    [OUTPUT]
-        Name hsdp
-        Match kube.*
-
-    [OUTPUT]
-        Name hsdp
-        Match host.*
-
-  ## https://docs.fluentbit.io/manual/pipeline/parsers
-  customParsers: |
-    [PARSER]
-        Name docker_no_time
-        Format json
-        Time_Keep Off
-        Time_Key time
-        Time_Format %Y-%m-%dT%H:%M:%S.%L
-
-  # This allows adding more files with arbitary filenames to /fluent-bit/etc by providing key/value pairs.
-  # The key becomes the filename, the value becomes the file content.
-  extraFiles: {}
-#     example.conf: |
-#       [OUTPUT]
-#           Name example
-#           Match foo.*
-#           Host bar
-
-# The config volume is mounted by default, either to the existingConfigMap value, or the default of "fluent-bit.fullname"
-volumeMounts:
-  - name: config
-    mountPath: /fluent-bit/etc/fluent-bit.conf
-    subPath: fluent-bit.conf
-  - name: config
-    mountPath: /fluent-bit/etc/custom_parsers.conf
-    subPath: custom_parsers.conf
-
-daemonSetVolumes:
-  - name: varlog
-    hostPath:
-      path: /var/log
-  - name: varlibdockercontainers
-    hostPath:
-      path: /var/lib/docker/containers
-  - name: etcmachineid
-    hostPath:
-      path: /etc/machine-id
-      type: File
-
-daemonSetVolumeMounts:
-  - name: varlog
-    mountPath: /var/log
-  - name: varlibdockercontainers
-    mountPath: /var/lib/docker/containers
-    readOnly: true
-  - name: etcmachineid
-    mountPath: /etc/machine-id
-    readOnly: true
-
-args: []
-
-command: []
-
-# This supports either a structured array or a templatable string
-initContainers: []
-
-# Array mode
-# initContainers:
-#   - name: do-something
-#     image: bitnami/kubectl:1.22
-#     command: ['kubectl', 'version']
-
-# String mode
-# initContainers: |-
-#   - name: do-something
-#     image: bitnami/kubectl:{{ .Capabilities.KubeVersion.Major }}.{{ .Capabilities.KubeVersion.Minor }}
-#     command: ['kubectl', 'version']
-
-logLevel: info
+fluent-bit:
+  extraVolumes:
+    - name: plugins
+      emptyDir: {}
+  extraVolumeMounts:
+    - mountPath: /out
+      name: plugins
+  initContainers:
```

Comment:
can't we use the extra containers? so we don't have to copy the file?

containers in a pod can share the volumes.

Reply:
But then we would need to keep that sidecar pod runningjust doing a sleep loop forever? In that case the current `initContainer` feels more economical.